src/main/java/org/candlepin/subscriptions/BootApplication.java

```diff
@@ -20,16 +20,21 @@
  */
 package org.candlepin.subscriptions;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.autoconfigure.quartz.QuartzAutoConfiguration;
 
 /** Bootstrapper for Spring Boot. */
-@SpringBootApplication(exclude = { QuartzAutoConfiguration.class }) // we manually import in scheduler profile
+@SpringBootApplication
 @SuppressWarnings("checkstyle:hideutilityclassconstructor")
 public class BootApplication {
+    private static final Logger log = LoggerFactory.getLogger(BootApplication.class);
```

Comment:
Unused, please remove (along with imports) - this is what SonarCloud analysis flagged.